### PR TITLE
fix: healthchecks failing due to missing internal id

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -8,4 +8,4 @@ else
   export LLAMA_N_CTX=8182
 fi
 
-poetry run python -m uvicorn skynet.main:app --reload
+poetry run python -m skynet.main
```
Comment (saghul) [inline run.sh:11]: Why the change?

**Review by saghul — APPROVED** [review]

File: run.sh (hunk 1 above)
Comment (quitrk) [reply run.sh:11]: to mirror what we are running the deployment, currently it was bypassing the normal startup flow from the `main` module

Comment (saghul) [reply run.sh:11]: aha!
